.travis.yml

```diff
@@ -43,6 +43,7 @@ matrix:
   - env:
       - DISPLAY=':99.0'
       - TESTING_INTERPRETER=yes
+      - TESTING_AST_ANALYSIS=yes
```

Comment:
This test case only runs the ActionCable subscription tests from `spec/dummy`, did you mean to set this flag for those tests? I don't see the flag used by those tests.